packages/editor/src/client/blocks/embeds/iframe.tsx

```diff
@@ -41,7 +40,9 @@ export function IframeEmbed({url, title, width, height, styleCustom, sandbox}: I
           scrolling="no"
           frameBorder="0"
           allowFullScreen
-          sandbox={sandbox}
+          width="100%"
+          height={height}
+          sandbox={sandbox?.length !== 0 ? sandbox : undefined}
```

Comment:
U can write this shorter, because 0 is a falsy value.

`sandbox={!!sandbox?.length ? sandbox : undefined}`